app/[locale]/template/[templateId]/history/page.tsx

```diff
@@ -64,72 +68,78 @@ const TemplateHistory = () => {
-                            <div data-testid="author">{`by ${lastPublication?.versionedBy?.affiliation?.displayName}`}</div>
-                            <div>
-                                <span data-testid="latest-version" className={styles.historyVersion}>Version {lastPublication?.version.slice(1)}</span>
-                                <span data-testid="publication-date">Published: {lastPublicationDate}</span>
-                            </div>
-                        </div>
-                    </>
-                )}
-
-                <h2 id="templateHistoryHeading">History</h2>
-                <Table aria-labelledby="templateHistoryHeading" className="react-aria-Table">
-                    <TableHeader className="react-aria-TableHeader">
-
-                        <Column isRowHeader={true} className="react-aria-Column">Action</Column>
-                        <Column isRowHeader={true} className="react-aria-Column">User</Column>
-                        <Column isRowHeader={true} className="react-aria-Column">Time and Date</Column>
-                    </TableHeader>
-                    <TableBody>
-                        {
-                            sortedTemplates.length > 0
-                                ? sortedTemplates.map((item, index) => {
-
-                                    const publishDate = item?.created ? formatWithTimeAndDate(item?.created) : '';
-                                    const versionedBy = item?.versionedBy;
-
-                                    return (
-                                        <Row key={`${item?.id}-${index}`} className="react-aria-Row">
-                                            <Cell className="react-aria-Cell">
-                                                <div>Published {item?.version}</div>
-                                                <div>
-                                                    <small className={styles.changeLog}>
-                                                        Change log:<br />{item?.comment}
-                                                    </small>
-                                                </div>
-                                            </Cell>
-                                            <Cell className="react-aria-Cell">
-                                                {versionedBy
-                                                    ? `${versionedBy.givenName || ''} ${versionedBy.surName || ''}`
-                                                    : 'Unknown'}</Cell>
-                                            <Cell>{publishDate}</Cell>
-                                        </Row>
-                                    );
-                                })
-                                : <Row><Cell>No template history available.</Cell></Row>
-                        }
-
-                    </TableBody>
-                </Table>
+                <LayoutWithPanel>
```

Comment:
`LayoutWithPanel` is extra overhead (scripts and react hooks and effects) and provides CSS for sidebar or drawer panels. Using it without either a `SidebarContainer` or a `DrawerContainer` isn't ideal. Just use `LayoutContainer` if there are no sidebars ;)

Reply:
This might be why you had layout issues with the sections ?